rust/src/coordinator.rs

```diff
@@ -19,6 +19,41 @@ use super::{
     PetError,
 };
 
+/// A public key generated by a sum participant. It is used by the
+/// update participants to encrypt their masking seed for each sum
+/// participant.
+type SumParticipantEphemeralPublicKey = box_::PublicKey;
+
+/// A public key that identifies a sum participant.
+type SumParticipantPublicKey = box_::PublicKey;
+
+/// Dictionary created during the first phase of the protocol. It maps
+/// the public key of every sum participant to the ephemeral public
+/// key that participant generated.
+type SumDict = HashMap<SumParticipantPublicKey, SumParticipantEphemeralPublicKey>;
+
+/// Public key that identifies an update participant
+type UpdateParticipantPublicKey = box_::PublicKey;
+
+/// Seed dictionaries are sent by update participants. They contain
+/// the participant's seed, encrypted with the public key of each sum
```

Comment:
`encrypted with the ephemeral public key` (the distinction between the public key and the ephemeral public key is important here)

Reply:
oops good catch, thanks